### ISSUE TITLE
How to programatically set an input value after it changed

### ISSUE THREAD

**Issue by lucasbento** [rojobuffalo/formsy-material-ui#136]
First of all, thank you for this, really helped me.

Is there any way to change the value of an input when user fulfills it?

I'm trying the following and it is not working:

``` javascript
maskInput = (event) => {
  const { handleCentimeters } = this.props;
  this[event.target.name].setValue(handleCentimeters(event.target.value));
  // event.target.value = handleCentimeters(event.target.value); does not work either

  console.log(this[event.target.name]); // prints the exactly changed input
}
```

``` javascript
<Formsy.Form
   onChange={this.validateForm}
   onValid={this.validForm}
   onInvalid={this.invalidForm}
   onValidSubmit={this.submitForm}
>
   <FormsyText
     name="width"
     validations="isMinimumWidth"
     ref={ref => this.width = ref}
     validationError={minimumWidthError}
     style={{ ...styles.textField, marginRight: 10 }}
     hintText="Largura (cm)"
     floatingLabelText="Largura"
     required={true}
     onChange={this.maskInput}
   />
</Formsy.Form>
```

**Comment by lucasbento** [comment]
I was able to do this using `<FormsyText />`'s `value` prop, e.g.:

```javascript
maskInput = (event) => {
  const { handleCentimeters } = this.props;

  this.setState({
    ...this.state,
    [event.target.name]: handleCentimeters(event.target.value),
  });
}
```

```javascript
<FormsyText
  name="width"
  validations="isMinimumWidth"
  validationError={minimumWidthError}
  hintText="Largura (cm)"
  onChange={this.maskInput}
  value={this.state.width.centimeters}
/>
```

But I still have a feeling that this should be handled by formsy-react.

**Comment by rojobuffalo** [comment]
Closed by #220
